client/components/ReturnChatQueueModal.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, Button, ButtonGroup, Icon, Modal } from '@rocket.chat/fuselage';
+import React, { FC } from 'react';
+
+import { useTranslation } from '../contexts/TranslationContext';
+import { withDoNotAskAgain, RequiredModalProps } from './withDoNotAskAgain';
+
+type ReturnChatQueueModalProps = RequiredModalProps & {
+	onMoveChat: () => void;
+	onCancel: () => void;
+};
+
+const ReturnChatQueueModal: FC<ReturnChatQueueModalProps> = ({
+	onCancel,
+	onMoveChat,
+	confirm = onMoveChat,
+	dontAskAgain,
+	...props
+}) => {
+	const t = useTranslation();
+
+	return <Modal {...props}>
+		<Modal.Header>
+			<Icon name='burger-arrow-left' size={20}/>
```

Comment:
I think we should find a better place to host this specific components/modals they dont make sense at 'rocket.chat' level. only at omnichannel, I dont think we should let them at the component 'root' level. People will not use this component for a different case.